src/ott/neural/data/semidiscrete_dataloader.py

```diff
@@ -32,24 +32,30 @@ class SemidiscreteDataloader:
   ``(source, target)`` arrays of shape ``[batch, ...]``.
 
   Args:
-    rng: Random number seed.
+    rng: Random number seed used for sampling from the source distribution.
     sd_out: Semidiscrete output object storing a precomputed OT solution between
       the (continuous) source distribution and the dataset of interest.
     batch_size: Batch size.
+    epsilon: Epsilon regularization. If :obj:`None`, use the one stored
```

Comment:
epsilon: Epsilon regularization. In the context of this class, this epsilon value can be interpreted exclusively as a soft-max temperature.  If :obj:`None`, use the one stored in the :attr:`geometry <ott.solvers.linear.semidiscrete.SemidiscreteOutput.geom>` which was used to compute the potential stored in ``sd_out``, which is the sound thing to do.